### PR TITLE
Refactor enums

### PR DESCRIPTION
**Pull request by Ranray**

### CODE REVIEW

File: Soccar [Library]/src/nl/soccar/library/enumeration/ThrottleAction.java [hunk 1]
```diff
@@ -7,6 +7,31 @@
...
+
+    private ThrottleAction(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static ThrottleAction parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/ThrottleAction.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/SteerAction.java [hunk 2]
```diff
@@ -7,6 +7,31 @@
...
+    
+    private SteerAction(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static SteerAction parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/SteerAction.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/HandbrakeAction.java [hunk 3]
```diff
@@ -2,11 +2,33 @@
...
-    ACTIVE, INACTIVE
+    private HandbrakeAction(int id) {
+        this.id = id;
+    }
     
+    public int getId() {
+        return id;
+    }
+    
+    public static HandbrakeAction parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/HandbrakeAction.java:23]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

File: Soccar [Library]/src/nl/soccar/library/enumeration/MapType.java [hunk 4]
```diff
@@ -7,8 +7,31 @@
...
+
+    private MapType(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static MapType parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/MapType.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/Privilege.java [hunk 5]
```diff
@@ -7,8 +7,31 @@
...
+
+    private Privilege(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static Privilege parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/Privilege.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/Duration.java [hunk 6]
```diff
@@ -7,8 +7,31 @@
...
+
+    private Duration(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static Duration parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/Duration.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/TeamColour.java [hunk 7]
```diff
@@ -7,7 +7,28 @@
...
+
+    private TeamColour(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static TeamColour parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/TeamColour.java:23]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/BallType.java [hunk 8]
```diff
@@ -7,8 +7,31 @@
...
+
+    private BallType(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static BallType parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/BallType.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Library]/src/nl/soccar/library/enumeration/CarType.java [hunk 9]
```diff
@@ -7,8 +7,31 @@
...
+
+    private CarType(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static CarType parse(int id) {
```
Comment (ghost) [inline Soccar [Library]/src/nl/soccar/library/enumeration/CarType.java:24]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 9 issues
* MINOR 9 minor

Watch the comments in this conversation to review them.

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**